src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -50,15 +50,74 @@ public enum GCNotificationStatus
         NotApplicable = 4
     }
 
+    public struct GCMemoryInfo
+    {
+        /// <summary>
+        /// HighMemoryLoadThreshold when the last GC occured
+        /// </summary>
+        public int HighMemoryLoadThreshold { get; }
+
+        /// <summary>
+        /// Total available memory for the GC to use when the last GC ocurred. By default this is the physical memory on the machine, but it may be customized by specifying a HardLimit.
+        /// </summary>
+        public long TotalAvailableMemory { get; }
+
+        /// <summary>
+        /// Memory Load when the last GC ocurred
+        /// </summary>
+        public int MemoryLoad { get; }
```

Comment:
What is the value returned from this?  Is it a percentage?  It's not particularly clear from the name or the return type.

Reply:
it's a percentage. any suggestion what a better name could be?

Reply:
> any suggestion what a better name could be?

MemoryLoadPercentage?

Is this a value between 0 and 100?  Should it be a double, and if so, between 0-1 vs 0-100?

Reply:
@stephentoub @Maoni0 a similar discussion happened on https://github.com/dotnet/corefx/issues/34631, and I will make sure for them to be discussed during tomorrow's API design review meeting.